### PR TITLE
feat: help route with questions and answers

### PR DESCRIPTION
**Pull request by nac62116**

### CODE REVIEW

File: app/routes/__components.tsx [hunk 1]
```diff
@@ -1043,13 +1046,10 @@ function Icon(props: { type: IconType }) {
         fill="none"
         xmlns="http://www.w3.org/2000/svg"
       >
```
Comment (nac62116) [inline app/routes/__components.tsx:1048]: Add new outline icon

File: app/routes/help.tsx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    <>
+      <section className="mv-w-full mv-mx-auto @md:mv-max-w-screen-container-md @lg:mv-max-w-screen-container-lg @xl:mv-max-w-screen-container-xl @2xl:mv-max-w-screen-container-2xl mv-px-4 @md:mv-px-6 @xl:mv-px-8 mv-pt-16 mv-pb-8 @md:mv-pb-12 @xl:mv-pb-16">
+        <h1 className="mv-w-full mv-text-center mv-text-5xl @sm:mv-text-6xl @md:mv-text-7xl @xl:mv-text-8xl mv-font-[900] mv-leading-9 @sm:mv-leading-10 @md:mv-leading-[64px] @xl:mv-leading-[80px]">
+          {t("headline")}
+        </h1>
+        <p className="mv-w-full mv-text-center mv-text-neutral-700 mv-leading-5">
+          {t("subline")}
+        </p>
+      </section>
+      {/* Margin bottom in this section is only applied because the next secion is not yet implemented (Support CTA) */}
```
Comment (nac62116) [inline app/routes/help.tsx:23]: Outsource to __help.components.tsx

File: app/routes/help.tsx [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+      <section className="mv-w-full mv-mx-auto @md:mv-max-w-screen-container-md @lg:mv-max-w-screen-container-lg @xl:mv-max-w-screen-container-xl @2xl:mv-max-w-screen-container-2xl mv-px-4 @md:mv-px-6 @xl:mv-px-8 mv-py-6 mv-mb-[72px]">
```
Comment (nac62116) [inline app/routes/help.tsx:24]: Give section an id and @apply tailwind styles to <ul>, <p>, <a> and <li>

**Review by phollome — APPROVED** [review]
🚀

**Pull request merged**